apps/eth-wrapper/src/assets/svg/weth.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="2000" height="2000" viewBox="0 0 2000 2000"><rect x="184.16" width="1815.84" height="1815.84" rx="267.39" fill="#939393"/><rect y="184.16" width="1815.84" height="1815.84" rx="267.39" fill="#303030"/><path d="M907.79,367.78l-9.71,33v957.83l9.71,9.69,444.59-262.79Z" fill="#9b9b9b"/><path d="m907.79 367.78l-444.59 737.74 444.59 262.81" fill="#fff"/><path d="M907.79,1452.52l-5.48,6.68v341.25l5.48,15.93,444.85-626.55Z" fill="#9b9b9b"/><path d="m907.79 1816.38l0-363.87-444.59-262.68" fill="#fff"/><path d="M907.79,1368.33l444.58-262.81L907.79,903.43v464.9" fill="#666"/><path d="m463.2 1105.52l444.58 262.81 0-464.9" fill="#9b9b9b"/></svg>
```

Comment:
This is not really the official WETH icon, right? (If it is than the app icon should also be this one)

Reply:
ah yeah, I don't remember exactly, where I got that from, but I think I ended up using the one returned for weth from our services `TokenInfo`. I will get rid of it :)